crates/cli/src/flags.rs

```diff
@@ -226,6 +226,12 @@ fn get_start_command() -> Command {
                 .default_value("true")
                 .default_missing_value("true"),
         )
+        .arg(
+            arg!(--"request-buffer-size" <BYTES>)
+            .help("The buffer size of the stream that is used to forward a request to the worker")
+            .value_parser(value_parser!(u64))
+            .default_value("16384"),
```

Comment:
I noticed when the flag isn't set, server defaults to 1KiB. Should that be changed to 16KiB as well?

Reply:
> I noticed when the flag isn't set, server defaults to 1KiB. Should that be changed to 16KiB as well?

I don't think there is a good default value for this, all I can tell you is that making it too small will increase the amount of wasted user worker CPU time because the buffer will have to be emptied from time to time if the request is too large.

Reply:
Prior to this PR, this value was fixed at 1024 bytes, and I suspect that the amount of CPU time lost when requests with very large payloads were sent was not small.